libs/eo/metering-points/shell/src/lib/eo-metering-points-shell.component.ts

```diff
@@ -20,27 +20,59 @@ import { takeUntilDestroyed } from '@angular/core/rxjs-interop';
 
 import { WATT_CARD } from '@energinet-datahub/watt/card';
 import { WattToastService } from '@energinet-datahub/watt/toast';
+import { WattValidationMessageComponent } from '@energinet-datahub/watt/validation-message';
 
 import { EoMeteringPointsTableComponent } from './eo-metering-point-table.component';
 import { EoMeteringPointsStore } from './eo-metering-points.store';
-import { EoBetaMessageComponent } from '@energinet-datahub/eo/shared/atomic-design/ui-atoms';
 
 @Component({
   changeDetection: ChangeDetectionStrategy.OnPush,
   standalone: true,
-  imports: [AsyncPipe, NgIf, EoMeteringPointsTableComponent, WATT_CARD, EoBetaMessageComponent],
+  imports: [
+    AsyncPipe,
+    NgIf,
+    EoMeteringPointsTableComponent,
+    WATT_CARD,
+    WattValidationMessageComponent,
+  ],
   selector: 'eo-metering-points-shell',
   styles: [
     `
-      watt-card-title {
+      :host {
         display: flex;
-        gap: var(--watt-space-m);
-        align-items: center;
+        flex-direction: column;
+        gap: var(--watt-space-l);
       }
     `,
   ],
   template: `
-    <eo-eo-beta-message></eo-eo-beta-message>
+    <watt-validation-message
+      label="You have the option to switch your metering points ON and OFF."
+      size="normal"
+      icon="info"
+      [autoScrollIntoView]="false"
+    >
+      <p>For Production metering points:</p>
+
+      <ul>
+        <li>Turning it 'ON' means the metering point is actively issuing certificates of power.</li>
+        <li>Switching it 'OFF' will stop the metering point from issuing certificates.</li>
+      </ul>
+
+      <br />
+      <p>For Consumption metering points:</p>
+
+      <ul>
+        <li>
+          'ON' signifies that the metering point is set to use electricity generated by another
+          Production unit that you have an agreement with (through this system).
```

Comment:
This is not exactly what will happen. 'ON' signifies that certificates will be generated for this metering point along and added to the user's wallet. Furthermore, claiming will be enabled in your wallet. But the claiming process just claims what is possible; it does not care about the exact "production unit" being used and it does not even care if the production certificate is from one of your metering points.